scripts/README.md

```diff
@@ -121,17 +121,13 @@ Flags:
 
 Compile and outputs the templates. For now, only README is being built and outputted.
 
-## test
+## bazel:test
```

Comment:
Should remove `packages/ngtools/webpack/src/type_checker_bootstrap.js` when removing the old test setup as mentioned in https://github.com/angular/angular-cli/pull/17304#discussion_r398091282.

Also should remove the package.json test scripts for it.